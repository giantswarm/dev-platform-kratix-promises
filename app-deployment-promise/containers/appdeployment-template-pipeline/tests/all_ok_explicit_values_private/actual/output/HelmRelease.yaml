apiVersion: helm.toolkit.fluxcd.io/v2beta1
kind: HelmRelease
metadata:
  name: my-go-project-1
  namespace: default
spec:
  interval: 1m
  chart:
    spec:
      chart: github-repo
      version: ">0.1.0"
      sourceRef:
        kind: HelmRepository
        name: flux-helm-repository
      verify:
        provider: cosign
  suspend: false
  timeout: "3m"
  values:
    imagePullSecrets:
      - name: ghcr-pull-secret
    enableDb: true
  dependsOn:
    - test1
  kubeConfig:
    secretRef:
      name: stage-kubeconfig
  valuesFrom:
    - kind: Secret
      name: prod-tls-values
      valuesKey: crt
      targetPath: tls.crt
      optional: true
