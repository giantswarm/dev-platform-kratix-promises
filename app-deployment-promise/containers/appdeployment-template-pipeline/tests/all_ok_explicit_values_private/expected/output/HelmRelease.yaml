apiVersion: helm.toolkit.fluxcd.io/v2beta1
kind: HelmRelease
metadata:
  annotations:
    kustomize.toolkit.fluxcd.io/ssa: ${my_go_project_1:=Ignore}
  name: my-go-project-1
  namespace: default
  labels:
    giantswarm.io/cluster: demotech-rds
spec:
  interval: 1m
  chart:
    spec:
      chart: my-go-project-1
      version: ">0.1.0"
      sourceRef:
        kind: HelmRepository
        name: my-go-project-1
        # Disable until Flux v2.2.0 cos keyless verification is not supported before that
        # or support key based signing: https://gigantic.slack.com/archives/C02GDJJ68Q1/p1723710023444289?thread_ts=1722593976.383529&cid=C02GDJJ68Q1
  #      verify:
  #        provider: cosign
  suspend: false
  timeout: "3m"
  values:
    autoscaling:
      enabled: false
    database:
      connectionSecretName: my-go-project-1-rds-user
      name: my-go-project-1
    inMemoryStore: false
    ingress:
      annotations:
        kubernetes.io/tls-acme: "true"
      enabled: true
      host: demo.tech.none
      tls:
        - hosts:
            - demo.tech.none
          secretName: my-go-project-1
    monitoring:
      serviceMonitor:
        enabled: true
        extraLabels:
          application.giantswarm.io/team: my-team
    pdb:
      enabled: false
    imagePullSecrets:
      - name: ghcr-pull-secret
    enableDb: true
    redisConnectionSecretName: my-go-project-1-elasticache-user
  dependsOn:
    - name: test1
  kubeConfig:
    secretRef:
      name: stage-kubeconfig
